### PR TITLE
Add more data about the Pontus-X ParaTime

### PR DESCRIPTION
**Pull request by csillag**
~This depends on #1265~.

### CODE REVIEW

File: src/app/components/LayerPicker/LayerDetails.tsx [hunk 1]
```diff
@@ -68,6 +68,14 @@ const getDetails = (t: TFunction): Details => ({
       chainDecimalId: '23295',
       docs: docs.sapphire,
     },
+    [Layer.pontusx]: {
+      description: t('layerPicker.testnet.pontusx'),
+      rpcHttp: 'https://pontusx.not.si:443',
+      // rpcWebSockets: 'wss://testnet.sapphire.oasis.dev/ws', // TODO: fix this
```
Comment (lubej) [inline src/app/components/LayerPicker/LayerDetails.tsx:74]: WS support is not even that important. Not sure if Sapphire wrapper even supports it(I could be wrong). The important one is RPC one.

Comment (csillag) [reply src/app/components/LayerPicker/LayerDetails.tsx:74]: Yes. It's not that important, that's why I made that an optional field in this description type, and updated the UI code to handle cases when it's missing. I think we can safely merge with this one field missing.

File: src/config.ts [hunk 2]
```diff
@@ -164,3 +173,12 @@ const stableDeploys = [...deploys.production, deploys.staging]
 export const isStableDeploy = stableDeploys.some(url => window.location.origin === url)
 
 export const getAppTitle = () => process.env.REACT_APP_META_TITLE
+
+export const getTickerForScope = ({ network, layer }: SearchScope): NativeTicker => {
+  const networkDefault = getTickerForNetwork(network)
```
[the remark this answers is not in the document]
Comment (csillag) [reply src/config.ts]: It has already been removed as part of the underlying #1265 .

File: src/app/components/LayerPicker/LayerDetails.tsx (hunk 1 above)
Comment (lubej) [reply src/app/components/LayerPicker/LayerDetails.tsx:74]: I would simply delete this line.

**Review by lubej — APPROVED** [review]
LGTM

File: src/app/components/LayerPicker/LayerDetails.tsx (hunk 1 above)
Comment (csillag) [reply src/app/components/LayerPicker/LayerDetails.tsx]: Deleted.

**Pull request merged**